react-basic-practice/src/App.tsx

```diff
@@ -1,14 +1,15 @@
 import React from "react";
 import { Box, ChakraProvider } from "@chakra-ui/react";
 import { theme } from "./themes";
-import { Search } from "@components";
+import { Table } from "@components";
+import { columns } from "./constants/columns";
```

Comment:
Pls add `alias path`, create `index.ts` file for constants folder

Reply:
i done